incubator/aws-alb-ingress-controller/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Please remove the license file from the chart. Everything in this repository is under the [license file at the root](https://github.com/helm/charts/blob/master/LICENSE).